doc/widgets/geomap.rst

```diff
@@ -13,20 +13,36 @@ Signals
 
 -  **Data**
 
-Data set.
+   Data set.
 
 **Outputs**:
 
 -  **Corpus**
 
-Corpus instance.
+   A :ref:`Corpus` instance.
 
 Description
 -----------
 
-No description yet for this widget.
+**GeoMap** widget shows geolocations from textual (string) data. It finds mentions of geographic names (countries and capitals) and displays distributions (frequency of mentiones) of these names on a map. It works with any Orange widget that outputs a data table and that contains at least one meta attribute. The widget outputs selected data instances, that is all documents containing mentions of a selected country (or countries).
```

Comment:
`It works with any Orange widget that outputs a data table and that contains at least one meta attribute.` -> It works with any Orange widget that outputs a data table and that contains at least one **string** attribute.